CHANGELOG.md

```diff
@@ -8,6 +8,8 @@ This is the log of notable changes to EAS CLI and related packages.
 
 ### 🎉 New features
 
+- Handle new resource class names in `eas-json` and `eas-cli`. ([#1616](https://github.com/expo/eas-cli/pull/1616) by [@szdziedzic](https://github.com/szdziedzic))
```

Comment:
Make this more generic + reference my PRs (https://github.com/expo/eas-cli/pull/1615 https://github.com/expo/eas-cli/pull/1609)